docs/api/api/api.txt

```diff
@@ -405,6 +405,11 @@ XmlBody: none
 XmlResult: status
 
 
+GET /source/<project>/_keyinfo
+
+XmlResult: keyinfo
```

Comment:
@adrianschroeter couldn't it be little more descriptive? Which key are we talking about?